pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
+
+`In> {{Integrate(x)Cos(x)}};`
```

Comment:
What is the In>

is that a part of the command you enter or are you trying to reference the fact that it is when you are in the repl

Reply:
Please see the pull-request note. "In>" is the prompt of the repl. I had also included "Out>" and the corresponding sample output but lint rejected it as more than one command. Do you have an example of how tldr represents a repl? I couldn't think of one. I tried "In> `statement;`" with the prompt outside the `` but my tldr client rendered it wrong.

I'm also unsure about the ";" required to terminate the statement. On second thought, it probably belongs inside the {{}} as part of the sample statement.

I've just pushed a commit that uses {{}} to distinguish the prompt from the user's input. It seems clearer to me.

Reply:
Normally we don't include any prompts, but I don't see any reason why not if you can clearly distinguish between them.